contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -6778,6 +6778,8 @@ void process_execsql_remove_unsupported_tokens(TSqlParser::Dml_statementContext
 static void
 post_process_column_constraint(TSqlParser::Column_constraintContext *ctx, PLtsql_stmt_execsql *stmt, TSqlParser::Ddl_statementContext *baseCtx)
 {
+	if (ctx->UNIQUE())
+		rewritten_query_fragment.emplace(std::make_pair(ctx->UNIQUE()->getSymbol()->getStopIndex()+1 , std::make_pair("", " NULLS NOT DISTINCT")));
```

Comment:
What are the implications on explain and DDL export?

Reply:
DDL export from TDS endpoint will remain unaffected, since nulls_not_distinct is a PG feature. No bbf catalog have this information.
From psql endpoint i.e. pg_dump will will start dumping the ddl for index/constraints with the NULLS NOT DISTINCT  keyword.

confirmed with explain (simple query on table with unique index) that queries will use the index/ constraint's index whenever possible. The NULLS FRIST/LAST fix in this PR fixes this for constraints.

Reply:
We also need to fix the pg_dump for constraints in babelfish.
Currently the ordering is not dumped. So we get something like this for constraints.
`ALTER TABLE... ADD CONSTRAINT ..... (col1, col2)`
NULLS ORDERING & ASC/DESC are not included. So if user does a dump & restore or MVU then ordering information is lost which means some queries would not be able to use the constraints index.